EIPS/eip-draft_title_abbrev.md

```diff
@@ -0,0 +1,74 @@
+
+`scheme://registry_address:chain_id/package_name@package_version/json_pointer`
+
+#### `scheme`
+- Required
+- Must be one of `ethpm` or `erc2678`. If future versions of the EthPM packaging standard are designed and published via the ERC process, those ERCs will also be valid schemes.
+
+#### `registry_address`
+- Required
+- This **should** be either an ENS name or a 0x-prefixed, checksummed address. ENS names are more suitable for cases where mutability of the underlying asset is acceptable and there is implicit trust in the owner of the name. 0x prefixed addresses are more preferable in higher security cases to avoid needing to trust the controller of the name.
+
+#### `chain_id`
+- Optional
+- Integer representing the chain id on which the registry is located
+- If omitted, defaults to `1` (mainnet).
+
+#### `package_name`
+- Optional
+- String of the target package name
+
+#### `package_version`
+- Optional
+- String of the target package version
+- If the package version contains any [url unsafe characters](https://www.werockyourweb.com/url-escape-characters/), they **must** be safely escaped
+- Since semver is not strictly enforced by the ethpm spec, if the `package_version` is omitted from a uri, tooling **should** avoid guessing in the face of any ambiguity and present the user with a choice from the available versions.
+
+#### `json_pointer`
+- Optional
+- A path that identifies a specific asset within a package.
+- This path **MUST** conform to the [JSON pointer](https://tools.ietf.org/html/rfc6901) spec and resolve to an available asset within the package.
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+Most interactions within the EthPM ecosystem benefit from a single-string representation of EthPM assets; from installing a package, to identifying a registry, to distributing a package. A single string that can faithfully represent any kind of EthPM asset, across the mainnet or testnets, reduces the mental overload for new users, minimizes configuration requirements for frameworks, and simplifies distribution of packages for package authors.
+
+## Test Cases
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+A JSON file for testing various URIs can be found in the [`ethpm-spec`](https://github.com/ethpm/ethpm-spec) repository.
```

Comment:
Any chance this could be inlined or added as an asset in `../assets/eip-2942/whatever.json`?

Reply:
Sure, but in this case I'd prefer to link towards the fixtures in the `ethpm-spec` repository to maintain parity between the spec and EIP should the fixtures be updated / renamed / etc. Rather than manage multiple "sources of truth".